e2e_tests/tests/experiment/test_allocation_csv.py

```diff
@@ -48,15 +48,20 @@ def test_notebook_capture() -> None:
 
```

Comment:
Looks like on line 36, `int(row["experiment_id"]` is resulting an error in the test because it's now sending an empty string instead of 0.

Reply:
good catch! I think the line 36 one is actually ok because we don't have any tasks that cause that case to come up. The failure was in `test_tensorboard_experiment_capture`. In any case, I fixed both instances!